apps/emqx_authentication/test/emqx_authentication_SUITE.erl

```diff
@@ -28,6 +28,7 @@ all() ->
     emqx_ct:all(?MODULE).
 
 init_per_suite(Config) ->
+    application:set_env(ekka, strict_mode, true),
```

Comment:
unset it in end_per_suite? will this flag effect other suites?

Reply:
It's ok to set this setting globally in all the testsuites, because it's meant to crash transactions that touch the tables outside the shard.